frontend/src/tests/GrantApplicationAdmin.spec.ts

```diff
@@ -0,0 +1,176 @@
+import router, { routes } from '@/router';
+import { DOMWrapper, mount, VueWrapper } from '@vue/test-utils';
+import { it, describe, beforeEach, expect, afterEach, vi } from 'vitest';
+import { routeItems } from '@/router/routeItem';
+import { fixJsdomCssErr } from './common/fixJsdomCssErr';
+import GrantApplicationAdmin from '@/components/grantaccess/GrantApplicationAdmin.vue';
+import waitForExpect from 'wait-for-expect';
+import { populateBreadcrumb } from '@/store/BreadcrumbState';
+
+fixJsdomCssErr();
+const mockRoutePush = vi.fn();
+vi.mock('vue-router', async () => {
+    const actual: Object = await vi.importActual('vue-router');
+    return {
+        ...actual,
+        useRoute: () => {
+            return routes.filter(
+                (route) => route.name == routeItems.grantAppAdmin.name
+            )[0];
+        },
+        useRouter: () => {
+            return {
+                push: mockRoutePush,
+            };
+        },
+    };
+});
+
+describe('GrantApplicationAdmin', () => {
+    let wrapper: VueWrapper;
+    let usernameInputText: DOMWrapper<HTMLElement>;
+    let usernameInputTextEl: HTMLInputElement;
+    let verifyButton: DOMWrapper<HTMLElement>;
+    let verifyButtonEl: HTMLButtonElement;
+    const routerPushSpy = vi.spyOn(router, 'push');
+
+    //populate the breadcrumbState
+    const breadcrumbItems = [routeItems.dashboard, routeItems.grantAppAdmin];
+    populateBreadcrumb(breadcrumbItems);
+    beforeEach(async () => {
+        wrapper = mount(GrantApplicationAdmin, {
+            global: {
+                plugins: [router], // Inject mocked Vue Router
+            },
+        });
+        usernameInputText = wrapper.find('#userIdInput');
+        usernameInputTextEl = usernameInputText.element as HTMLInputElement;
+        verifyButton = wrapper.find("[data-target-btn='verifyIdir']");
+        verifyButtonEl = verifyButton.element as HTMLButtonElement;
```

Comment:
Hi Jota, I'm not seeing we use the usernameInputText, usernameInputTextEl, verifyButton and verifyButtonEl in multiple test cases, I think we can remove them from here and the above.